be/src/common/config.h

```diff
@@ -505,6 +505,11 @@ CONF_String(es_scroll_keepalive, "5m");
 // HTTP connection timeout for es.
 CONF_Int32(es_http_timeout_ms, "5000");
 
+// es index max result window, and this value affects batch size.
+// if request batch size exceeds this value, ES will return bad request(400)
+// https://www.elastic.co/guide/en/elasticsearch/reference/current/index-modules.html
+CONF_Int32(es_index_max_result_window, "10000");
```

Comment:
I think it is better to make it as a const value rather than a configuration?
Because we don't want users to change this configure

Reply:
No, because different es maybe have different values.

Reply:
What's the default value for ES?

Reply:
default value is 10000

Reply:
<img width="799" alt="image" src="https://user-images.githubusercontent.com/8065637/168021702-3e61e1d3-7429-472f-b970-456ee4a9dc99.png">

Reply:
Thanks @DorianZheng That's fine for me.